spring-cloud-consul-discovery/src/test/java/org/springframework/cloud/consul/discovery/ConsulDiscoveryPropertiesTests.java

```diff
@@ -114,18 +114,6 @@ void getTagsReturnsMultipleFromServerListMapEntry() {
 		assertThat(properties.getQueryTagsForService(SERVICE_NAME_IN_MAP)).containsExactly("foo", "bar");
 	}
 
-	@Test
```

Comment:
Note that the StringUtils method does not remove whitespace. This is fine - being too nice w/ the property values is not worth the added code and users can just enter the values in w/o whitespace around entries.

Reply:
`tokenizeToStringArray` does that for you

Reply:
Ok I will check that out.